test/csuite/timestamp_abort/main.c

```diff
@@ -849,14 +859,14 @@ main(int argc, char *argv[])
                  */
                 c_rep[i].exist_key = key;
                 fatal = true;
-            } else if (!inmem && stable_fp != 0 && stable_fp > stable_val) {
+            } else if (!inmem && commit_fp != 0 && commit_fp > stable_val) {
```

Comment:
Compare the commit timestamp instead of durable timestamp with the stable timestamp here.